study.md

```diff
@@ -41,7 +41,9 @@ Does `proxima_centauri.print_distance` run? Why or why not? If so, then what is
 the output? If not, then how can it be fixed?
 
 ```md
-<!-- your answer here -->
```

Comment:
We're not passing any objects into the constructor.  `print_distance` is a method defined on the `Star` class.